lib/ClangImporter/ClangAdapter.cpp

```diff
@@ -563,6 +563,9 @@ OptionalTypeKind importer::translateNullability(clang::NullabilityKind kind) {
   case clang::NullabilityKind::Unspecified:
     return OptionalTypeKind::OTK_ImplicitlyUnwrappedOptional;
   }
+
+  // Work around MSVC warning: not all control paths return a value
+  llvm_unreachable("All switch cases are covered");
```

Comment:
You could argue this is good practice even outside of MSVC, since switch exhaustiveness isn't necessarily guaranteed covering all of an enum's constants (unless it happens to have exactly a power of two contiguously-valued elements).